src/map/script.c

```diff
@@ -14095,55 +14096,56 @@ static BUILDIN(setmapflag)
 	}
 	m = map->mapname2mapid(str);
 
-	if(m >= 0) {
-		switch(i) {
-			case MF_NOMEMO:             map->list[m].flag.nomemo = 1; break;
-			case MF_NOTELEPORT:         map->list[m].flag.noteleport = 1; break;
-			case MF_NOSAVE:             map->list[m].flag.nosave = 1; break;
-			case MF_NOBRANCH:           map->list[m].flag.nobranch = 1; break;
-			case MF_NOPENALTY:          map->list[m].flag.noexppenalty = 1; map->list[m].flag.nozenypenalty = 1; break;
-			case MF_NOZENYPENALTY:      map->list[m].flag.nozenypenalty = 1; break;
+	if (m >= 0) {
+		switch (i) {
+			case MF_NOMEMO:                  map->list[m].flag.nomemo = 1; break;
```

Comment:
Wrong indentation. Cases should have the same indentation as the `switch()`.

Reply:
But it's weird( mby change general typo rules to more c/c++ friendly? @Kenpachi2k13